@navikt/core/css/layout-container.css

```diff
@@ -0,0 +1,28 @@
+.navds-layout-container {
+  padding-left: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
+  padding-right: var(--ac-l-container-padding-xs, var(--navds-spacing-4));
+  margin-left: auto;
+  margin-right: auto;
+  max-width: var(--a-breakpoint-lg);
```

Comment:
Vil max-width til layout-container alltid være samme som breakpoint-lg, i alle apper? Jeg lurer på om det kan være fornuftig å definere en separat token for layout max-width.